app/komponenter/tekstBlokk/tekstBlokk.tsx

```diff
@@ -0,0 +1,44 @@
+import { TypografiTyper } from '~/typer/typografi';
+import React from 'react';
+import { PortableText } from '@portabletext/react';
+import { TypografiWrapper } from '~/utils/typografiWrapper';
+import css from './veilederhilsen.module.css';
+
+interface TekstBlokkProps {
+  tekstblokk: any | undefined;
+  valgBlock: string;
+  typografi?: TypografiTyper;
+}
+
+const TekstBlokk: React.FC<TekstBlokkProps> = ({
+  tekstblokk,
+  typografi,
+  valgBlock,
+}: TekstBlokkProps) => {
+  return tekstblokk ? (
+    <PortableText
+      value={tekstblokk[valgBlock]}
```

Comment:
Hvis `valgBlock` kun brukes for å indeksere `tekstblokk` kan dette skje utenfor komponenten. Da kan dere heller ta inn f.eks. `innhold: SanityDokument` som prop og slipper unna `any`

Reply:
Dette er ikke gjort noe med foreløpig men er notert :)